### DISCUSSION TITLE
Project / App Name

### DISCUSSION THREAD

**Discussion by alensiljak** [orgzly-revived/orgzly-android-revived#8] [General]
Opening for the name suggestions.

My suggestion is "Orgzly II".

**Comment by amberin** [comment]
I suggest: "Orgzly (Community Edition)"

(BTW, I have a branch where I tried changing the name, and the APK seems to work well.)

**Comment by camoz** [comment]
For the record, other suggestions from https://github.com/orgzly/orgzly-android/issues/1039 were:

- Orgzly Revived
- Orgzly Reboot

Another idea:

- Orgzly Fork

Out of all, I think my vote goes for "Orgzly Revived" since it's the most descriptive, or "Orgzly (Community Edition)" but that's a bit long and has special characters in it.

**Reply by alensiljak** [reply]
That gets easily shortened to "Orgzly CE".

**Reply by erikmd** [reply]
~~I vote down for~~ (→edit→) I vote against "_ CE" because it is often the counterpart of "_ EE" (Enterprise Edition, i.e., proprietary open source software)

**Reply by alensiljak** [reply]
> I vote down for

Do you mind clarifying / confirming what you mean by "vote down"? It is a bit ambiguous, especially for non-native speakers. 

https://www.wordnik.com/words/vote%20down

Edit: I agree that the CE is usually used to mean "a stripped-down version".

**Reply by amberin** [reply]
@camoz Would you mind putting those suggestions in separate posts, so that we can easily count their votes?

**Reply by camoz** [reply]
@amberin Yes I will!

**Reply by camoz** [reply]
@amberin We could also use this issue here to just gather ideas, and then create a Github Poll, which can be created under the Discussions tab.

**Reply by alensiljak** [reply]
The options in a poll cannot be modified, once set. It is OK for well-defined options but not very suitable for brainstorming.

**Comment by amberin** [comment]
> I vote down for "_ CE" because it is often the counterpart of "_ EE" (Enterprise Edition, i.e., proprietary open source software)

Good point!

**Comment by camoz** [comment]
Suggestion: "Orgzly Revived"

(from https://github.com/orgzly/orgzly-android/issues/1039)

**Comment by camoz** [comment]
Suggestion: "Orgzly Reboot"

(from https://github.com/orgzly/orgzly-android/issues/1039)

**Comment by camoz** [comment]
Suggestion: "Orgzly Fork"

**Comment by Itrekr** [comment]
Suggestion: Orgzly 2: Electric boogaloo

**Comment by Itrekr** [comment]
Other suggestions:

Orgzly the White
Uruk-zly
Orgzly Reincarnated
Orgzly 4 Ever
Orgzly the undying
Orgzly 3 (so they wonder what happened to 2)
Orgzly Plus
Orgzly Extreme
Better Orgzly
Neorgzly
Org-mode mobile
Orgzly Continued
Orgzly 2 Episode 2
O
App for usings of emacs org-mode on glorious mobile phoneture
Orgzly+
Sneedly (Formerly Org's)
Orgzly Dark
Lumpia
Orgzly Vanced

**Reply by edmundmiller** [reply]
I like  Neorgzly!

**Comment by markamaze** [comment]
Just a couple thoughts to consider for brainstorming:
- to me a shorter name is better
- should it include the original name 'Orgzly' that is just differentiated in some way?
- should the name be descriptive of its purpose in some way? such as including 'note' or something like that? 
- should it still include 'org' to tip off potential new users familiar with org-mode?



My suggestion: Org-anizer

**Reply by alensiljak** [reply]
I like this. But I'd try to avoid names that are too generic and difficult to search for. Adding or changing a letter would do. 
Organic and Organism also fall in that category.

**Reply by Itrekr** [reply]
While I do think a new name would be cool, previous orgzly users would probably search for orgzly instead of Orgaloomz or something else.

**Comment by camoz** [comment]
I think the name should be descriptive so users know what they get. With "Orgzly Revived", "Orgzly Reboot" and "Orgzly Community Edition (CE)" this is instantly clear.

Also I think it's nice to keep "Orgzly" in the name since that's what it is essentially, and it is discoverable by the existing userbase *and* the future userbase, given that there are quite some references to "Orgzly" around the internet, so people will probably continue to search for "Orgzly".

**Comment by alensiljak** [comment]
OK, thanks for contributions, everyone. I assume this is now resolved as a suggestion and a poll. The name is out and any modifications con be worked out through regular issues from here.

**Comment by 9z0** [comment]
Orgzly Revived is good for the purpose of getting the attention of the original user base and invokes the right impression.

But long term, I suggest a shorter and more independent name.

Since it is a fork, may I suggest "Forgzly" 😜 ?
